tests/feito/test_repository.py

```diff
@@ -0,0 +1,25 @@
+import os
+from unittest import TestCase
+from unittest import mock
+
+from git import Repo
+
+from feito import Repository
+
+# Note: always run this test case from the project root directory. It gets the actual repository
```

Comment:
It would be nice if we avoid such complexity. We can improve this by injecting `Repo` as a dependency and mocking it.

Reply:
Yes. True. I will alter `run.py` to the following:
```py
...
git_repo = Repo(os.getcwd())
repo = Repository(git_repo)
...
```
and `repository.py`

```py
..
def __init__(self, repo):
    self.repo = repo
...
```